### PR TITLE
Autoloading Classes

### PR DESCRIPTION
**Pull request by cerilacanda**

### CODE REVIEW

**Comment by cerilacanda** [comment]
@tbasilio Please review the content of `Autoloading Classes` lesson. Thank you.

**Review by clydiex — APPROVED** [review]

File: content/AutoloadingClasses/AutoloadingClasses.q.md [hunk 1]
```diff
@@ -0,0 +1,2860 @@
...
+        return $this->name;
+    }
+    private function checkAge($age)
+    {
+        if ($age > 0) {
+            return true;
+        }
+        return false;
+    }
+    abstract public function display();
```
Comment (rosoarv) [inline content/AutoloadingClasses/AutoloadingClasses.q.md]: can we add line spaces every after the methods?

File: content/AutoloadingClasses/AutoloadingClasses.q.md [hunk 2]
```diff
@@ -0,0 +1,2860 @@
...
+    public function getName()
+    {
+        return $this->name;
+    }
+    private function checkAge($age)
+    {
+        if ($age > 0) {
+            return true;
+        }
+        return false;
```
Comment (rosoarv) [inline content/AutoloadingClasses/AutoloadingClasses.q.md]: can we add line spaces in between each method?

File: content/AutoloadingClasses/AutoloadingClasses.q.md [hunk 3]
```diff
@@ -0,0 +1,2860 @@
...
+    }
+    abstract public function display();
```
Comment (rosoarv) [inline content/AutoloadingClasses/AutoloadingClasses.q.md]: add line space in between methods

**Review by rosoarv — COMMENTED** [review]
with regards to the file directory, can we have the following structure:
```
src/
    Class1.php
    Class2.php
    ...
includes/
   autoloader.php
```

**Comment by rosoarv** [comment]
And also, the `Autoload.php` should be `autoload.php`. This is based on what I usually encounter.

File: content/AutoloadingClasses/AutoloadingClasses.q.md [hunk 4]
```diff
@@ -307,7 +311,7 @@ spl_autoload_register("myAutoloader");
 
 ```php
 <?php
-require_once("./autoloader/Autoload.php");
+require_once(__DIR__ . "/includes/Autoload.php");
```
Comment (rosoarv) [inline content/AutoloadingClasses/AutoloadingClasses.q.md]: the `A` in `Autoload` is capital

File: content/AutoloadingClasses/AutoloadingClasses.q.md [hunk 5]
```diff
@@ -1110,7 +1129,7 @@ spl_autoload_register("myAutoloader");
 
 ```php
 <?php
-require_once("./Autoload.php");
+require_once(__DIR__ . "/Autoload.php");
```
Comment (rosoarv) [inline content/AutoloadingClasses/AutoloadingClasses.q.md]: here also

**Review by rosoarv — APPROVED** [review]

**Review by gjoren4 — APPROVED** [review]
Done reviewing. PR looks good.

**Pull request merged**